terraform/shared/elastic.tf

```diff
@@ -17,6 +17,18 @@ resource "ec_deployment" "catalogue_api" {
       size       = "8g"
       zone_count = 3
     }
+
+    config {
+      user_settings_yaml = yamlencode(
+        {
+          # This setting is a fix for performance problems with an optimisation added in 7.13.1
+          # See: https://www.elastic.co/guide/en/elasticsearch/reference/7.13/release-notes-7.13.2.html
+          # "Add setting to disable aggs optimization #73620 (issue: #73426)"
+          # TODO: A fix is likely in ES 7.14, after which we can remove this setting.
+          "search.aggs.rewrite_to_filter_by_filter" : true
```

Comment:
This won't make a difference really as:

> The order of precedence for cluster settings is:
>    * transient cluster settings
>    * persistent cluster settings
>    * settings in the elasticsearch.yml configuration file.

-- https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-update-settings.html#cluster-update-settings-api-desc